packages/storage-plus/src/indexes.rs

```diff
@@ -138,6 +138,15 @@ where
         )
     }
 
+    fn no_prefix(&self, p: K::NoPrefix) -> Prefix<T> {
```

Comment:
function makes sense, but let's remove the arg and replace `&p.prefix()` with `&[]`